tasks/41_sort-by-bits/sort-by-bits.js

```diff
@@ -0,0 +1,26 @@
+export function sortByBits(arrayOfNumbers) {
+  let arrayOfObjects = arrayOfNumbers.map((number) => {
```

Comment:
It would be better to avoid creating unnecessary objects (arrayOfObjects) and complexity.
Put the logic in compare function.
( tip: You can calculate numberOfOnes inside compare() )